internal/parser/parser_test.go

```diff
@@ -51,9 +51,10 @@ func BenchmarkParser(b *testing.B) {
 
 func testParser(runFn func(name string, fn func(t testing.TB))) {
 	for _, tt := range []struct {
-		name string
-		raw  string
-		want *ast.File
+		name      string
+		raw       string
+		want      *ast.File
+		nerrsWant int
```

Comment:
Can you rename this to just `nerrs`? I don't think the `want` is necessary.

Reply:
Sure, done.